api/types/container/host_config.go

```diff
@@ -370,6 +370,7 @@ type HostConfig struct {
 	// Applicable to UNIX platforms
 	CapAdd          strslice.StrSlice // List of kernel capabilities to add to the container
 	CapDrop         strslice.StrSlice // List of kernel capabilities to remove from the container
+	Capabilities    strslice.StrSlice // List of kernel capabilities to be available for container (this overrides the default set)
```

Comment:
This feature will land in API version v1.41. We need to make sure that this option is ignored when using an older version of the API. Similar to https://github.com/moby/moby/blob/852542b3976754f62232f1fafca7fd35deeb1da3/api/server/router/container/container_routes.go#L475-L479

So, when calling API < 1.41, `Capabilities` must be empty (or `nil` - see my other comment)

Reply:
Added and I also changed DefaultVersion to 1.41 so I was able to test this but I'm not sure if that should be part of this PR?